draft-ietf-jsonpath-base.md

```diff
@@ -1049,7 +1053,7 @@ true         = %x74.72.75.65                          ; true
 false        = %x66.61.6c.73.65                       ; false
 null         = %x6e.75.6c.6c                          ; null
 
-regex-expr   = (path / string-literal) S regex-op S regex
+regex-expr   = (singular-path / string-literal) S regex-op S regex
```

Comment:
Hmm, so I can't use a regexp to search in multiple elements.
I think that is something we can allow (~ matches at least one).

Reply:
I vote for the `singular-path` case here for clarity.

Remember ... filters are iterating over members or elements one by one and apply tests with each in turn ... so allowing multiple elements arising somehow would confuse ,,, at least me.

Reply:
Keeping `singular-path` on the LHS of regex expressions was the consensus as today's interim WG meeting.